AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,142 @@
+                                                                             style: configuration.style)
+
+    public var requiresModalPresentation: Bool = true
+
+    /// Component's configuration
+    public var configuration: Configuration
+
+    private let onlineBankingPaymentMethod: OnlineBankingPaymentMethod
+
+    private var selectedIssuer: Issuer
+
+    // MARK: - Items
+
+    private var termsAndConditionsLink: String {
+        return paymentMethod.type == .onlineBankingCZ ? "https://static.payu.com/sites/terms/files/payu_privacy_policy_cs.pdf" : "https://static.payu.com/sites/terms/files/payu_privacy_policy_sk.pdf"
+    }
+
+    /// The terms and condition message item.
+    internal lazy var termsAndConditionsLabelItem: FormAttributedLabelItem = .init(originalText:
+                        localizedString(.onlineBankingTermsAndConditions,
+                        configuration.localizationParameters),
+                        link: termsAndConditionsLink,
+                        style: configuration.style.footnoteLabel,
+                        linkTextStyle: configuration.style.linkTextLabel,
+                        identifier: ViewIdentifierBuilder.build(scopeInstance: self,
+                        postfix: ViewIdentifier.termsAndConditionsLabelItem))
+
+    /// The continue button item.
+    internal lazy var continueButton: FormButtonItem = {
+        let item = FormButtonItem(style: configuration.style.mainButtonItem)
+        item.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
+                                                      postfix: ViewIdentifier.continueButtonItem)
+        item.title = localizedString(.continueTitle, configuration.localizationParameters)
+        item.buttonSelectionHandler = { [weak self] in
+            self?.didSelectContinueButton()
+        }
+        return item
+    }()
+
+    /// The Issuer List item.
+    internal lazy var issuerListItem: FormIssuersPickerItem = {
+        let issuerListPickerItem: [IssuerPickerItem] = onlineBankingPaymentMethod.issuers.map({
+            IssuerPickerItem(identifier: $0.identifier, element: $0)
+        })
+        AdyenAssertion.assert(message: "Issuer list should not be empty", condition: issuerListPickerItem.count <= 0)
+
+        let issuerPickerItem = FormIssuersPickerItem(preselectedValue: issuerListPickerItem[0],
+                                        selectableValues: issuerListPickerItem,
+                                        style: configuration.style.textField)
+        issuerPickerItem.title = localizedString(.selectFieldTitle, configuration.localizationParameters)
+        issuerPickerItem.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
+                                                                  postfix: ViewIdentifier.issuerListItem)
+        observe(issuerPickerItem.publisher) { issuerPickerITem in
+            self.selectedIssuer = issuerPickerITem.element
+        }
```

Comment:
I don't think you need to observe it, just get the `issuerPickerItem.value` when submitting, right?